.circleci/config.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       - run:
           name: install python dependencies
           command: |
-            curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
+            curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python - --version=1.1.15
```

Comment:
This was required to pin the poetry version in the CI script because it was trying to install v1.2.1

I did try using the latest installer here but it didn't work, I wasn't able to work out why so have left it at 1.1.15 which is good enough.